tests/unit/test_utils.py

```diff
@@ -32,3 +40,35 @@ def async_return(result: t.Any) -> asyncio.Future:
     f = asyncio.Future()
     f.set_result(result)
     return f
+
+
+def mock_sqlalchemy(data: list = []) -> None:
```

Comment:
```suggestion
def mock_sqlalchemy(data: t.Optional[t.List[t.Any]] = None) -> None:
```
Using mutable default arguments like lists in Python functions can lead to unexpected behavior because the default argument is evaluated only once when the function is defined, not each time the function is called. This means that if you modify the list inside the function, the default list will retain those changes in subsequent calls. This can cause consequent bugs that are difficult to trace. So it's a common thing to make lists optional and then do smth like that:
```
if data is None:
    data = []
```

Reply:
Ah good to know.  That's so weird...

Reply:
I totally agree, that's a weird thing.:)